packages/block-library/src/template-part/editor.scss

```diff
@@ -77,6 +77,34 @@
 	}
 }
 
+.wp-block-template-part__label-container {
+	position: absolute;
+	bottom: 100%;
+	width: 100%;
+
+	.wp-block-template-part__label-layout {
+		display: flex;
+		justify-content: flex-start;
+	}
+
+	.wp-block-template-part__label-content {
+		display: flex;
+		align-items: center;
+		border: 1px solid $gray-600;
+		border-bottom: none;
+		border-top-right-radius: $radius-block-ui;
+		border-top-left-radius: $radius-block-ui;
+		padding: 0 6px;
+	}
+
+	.wp-block-template-part__label-text {
+		color: $black;
+		font-size: 10px;
```

Comment:
this font size seems a bit small, it might be good to use `$default-font-size` (which I believe is 13px).

There may be other places in this added css where it might make sense to use defined values from the base_styles package (https://github.com/WordPress/gutenberg/tree/master/packages/base-styles)

Reply:
That's a great callout. Thanks Addie! Making the changes now.